nx-X11/include/extensions/Imakefile

```diff
@@ -42,15 +42,6 @@ XINERAMAHEADERS = Xinerama.h panoramiXext.h panoramiXproto.h
 #if BuildRender || BuildXftLibrary
 RENDERHEADERS = render.h renderproto.h
 #endif
-#if BuildXfixes || BuildXfixesLibrary
-XFIXESHEADERS = xfixeswire.h xfixesproto.h
-#endif
-#if BuildDamage || BuildDamageLibrary
-DAMAGEHEADERS = damagewire.h damageproto.h
-#endif
-#if BuildComposite || BuildCompositeLibrary
-COMPOSITEHEADERS = composite.h compositeproto.h
-#endif
```

Comment:
No, please don't. I have been talking to keithp et. al.

If we're switching to system libraries, we should ALSO switch to system headers, including headers for the protocols.

Whatever we do, we're screwed anyway and will likely have to fix either subtle or not so subtle bugs.

Thus, it's better to completely base `damage`, `composite` and friends on the newer system libraries and headers.